test/integration/default/inspec/default_spec.rb

```diff
@@ -0,0 +1,7 @@
+describe http('http://localhost:4000/directory', enable_remote_worker: true) do
+  its('body') { should match(%r{"new-cert": "http://(localhost|127.0.0.1):4000/acme/new-cert"}) }
+end
+
+describe http('https://foo.org', enable_remote_worker: true, ssl_verify: false) do
+  its('status') { should cmp 403 }
```

Comment:
This should not be the status it's returning. The original status should be what you're seeing so please change this.

Reply:
The only status I saw (when I worked on this on Friday) was 403. When I login to the server and run `curl https://foo.org -k -v` I get a 403 status.

Reply:
Let me know if you want some help troubleshooting this.